src/modules/cookies.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Returns value for the specified cookie.
+ *
+ * @param {string} cname Cookie name.
+ * @returns {string}
+ */
+export function getCookie(cname) {
```

Comment:
Just a suggestion to rename it to cookieName

Reply:
Done. It was a reuse of code, so I didn't pay attention to naming :)